lib/config/migration.ts

```diff
@@ -39,14 +39,7 @@ export function migrateConfig(config: RenovateConfig): MigratedConfig {
       'peerDependencies',
     ];
     for (const [key, val] of Object.entries(newConfig)) {
-      if (key.startsWith('masterIssue')) {
-        const newKey = key.replace('masterIssue', 'dependencyDashboard');
-        migratedConfig[newKey] = val;
-        if (optionTypes[newKey] === 'boolean' && val === 'true') {
```

Comment:
Is this Boolean coercion already done elsewhere?

Reply:
Yes

Reply:
https://github.com/renovatebot/renovate/blob/c866fd06fc769494babedc70685a712696333730/lib/config/migration.ts#L145-L151